samples/connectedhomeip/lock/src/gen/endpoint_config.h

```diff
@@ -0,0 +1,158 @@
+/**
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *
+ *    Copyright (c) 2020 Silicon Labs
```

Comment:
Even worse :smile:

Reply:
I'm not quite sure what to do with the files generated by ZAP. To be honest they are Silicon Labs property and I don't think that we should change the copyrights, as we are not modifying them in any way.

Reply:
I think ZAP will eventually generate CHIP license headers, so probably we could use CHIP headers here, but the question is if it is OK with NCS code owners.

Reply:
But as I understand this does not happen yet. So most-likely we would need to use Silicon Labs license as is.. Let's see what will be the comments from the NCS community.